Safewaan/HonkedIn
# Style Events and My Events Page

## Trello ticket link

[Developer Task: Implement user feedback for UI/UX of Events Page](https://trello.com/c/vNs8Lr9k/134-developer-task-implement-user-feedback-for-ui-ux-of-events-page)



## Implementation description
* Styled Events and My Events Page
* Styled all dialogue options (View Event, Edit Event, Cancel Event, See Participants)



## Steps to test
1. Verify all dialogue options are working (ex: cancel an event, join an event, verify db is written to)
2. Verify there are no styling inconsistencies
3. Verify correct errors are thrown in error cases
